lib/code_corps_web/controllers/user_controller.ex

```diff
@@ -28,64 +28,71 @@ defmodule CodeCorpsWeb.UserController do
       conn |> render("show.json-api", data: user)
     end
   end
 
-  @spec create(Conn.t, map) :: Conn.t
+  @spec create(Conn.t(), map) :: Conn.t()
   def create(%Conn{} = conn, %{} = params) do
-    with {:ok, %User{} = user} <- %User{} |> User.registration_changeset(params) |> Repo.insert(),
-         user <- preload(user)
-    do
-      conn |> put_status(:created) |> render("show.json-api", data: user)
+    with {:ok, %User{} = user} <- params |> Accounts.create() do
+      conn
+      |> put_status(:created)
+      |> render("show.json-api", data: user |> preload())
     end
   end
 
-  @spec update(Conn.t, map) :: Conn.t
+  @spec update(Conn.t(), map) :: Conn.t()
   def update(%Conn{} = conn, %{"id" => id} = params) do
     with %User{} = user <- User |> Repo.get(id),
-         %User{} = current_user <- conn |> CodeCorps.Guardian.Plug.current_resource,
+         %User{} = current_user <- conn |> CodeCorps.Guardian.Plug.current_resource(),
          {:ok, :authorized} <- current_user |> Policy.authorize(:update, user),
          {:ok, user, _, _} <- user |> UserService.update(params),
-         user <- preload(user)
-    do
-       conn |> render("show.json-api", data: user)
+         user <- preload(user) do
+      conn |> render("show.json-api", data: user)
     end
   end
 
-  @doc """
+  @doc ~S"""
   Differs from other resources by path: `/oauth/github`
   """
-  @spec github_oauth(Conn.t, map) :: Conn.t
+  @spec github_oauth(Conn.t(), map) :: Conn.t()
   def github_oauth(%Conn{} = conn, %{"code" => code, "state" => state}) do
     current_user = Guardian.Plug.current_resource(conn)
+
     with {:ok, user} <- GitHub.API.User.connect(current_user, code, state),
-         user <- preload(user)
-    do
+         user <- preload(user) do
       Analytics.SegmentTracker.track(user.id, "Connected to GitHub", user)
       conn |> render("show.json-api", data: user)
     end
   end
 
-  @spec email_available(Conn.t, map) :: Conn.t
+  @spec email_available(Conn.t(), map) :: Conn.t()
   def email_available(%Conn{} = conn, %{"email" => email}) do
     hash = User.check_email_availability(email)
     conn |> json(hash)
   end
 
-  @spec username_available(Conn.t, map) :: Conn.t
+  @spec username_available(Conn.t(), map) :: Conn.t()
   def username_available(%Conn{} = conn, %{"username" => username}) do
     hash = User.check_username_availability(username)
     conn |> json(hash)
   end
 
   @preloads [
-    :categories, :github_app_installations, :organizations, :project_users,
-    :slugged_route, :stripe_connect_subscriptions, :stripe_platform_card,
-    :stripe_platform_customer, :user_categories, :user_roles, :user_skills
+    :categories,
+    :github_app_installations,
+    :organizations,
+    :project_users,
+    :slugged_route,
+    :stripe_connect_subscriptions,
+    :stripe_platform_card,
+    :stripe_platform_customer,
+    :user_categories,
+    :user_roles,
+    :user_skills
   ]
 
   def preload(data) do
```

Comment:
Another auto-formatting of file as I was modifying the `create` function. Actual changes are in the function above.